react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -1204,8 +1206,9 @@ function _mapStateToProps(state, ownProps): Object {
         _localFlipX: Boolean(localFlipX),
         _participant: participant,
         _raisedHand: hasRaisedHand(participant),
-        _stageFilmstripDisabled: !isStageFilmstripEnabled(state),
-        _stageParticipantsVisible: shouldDisplayStageFilmstrip(state, 1),
+        _stageFilmstripLayout: shouldDisplayStageFilmstrip(state, 0),
+        _stageParticipantsVisible: _currentLayout === LAYOUTS.STAGE_FILMSTRIP_VIEW,
```

Comment:
Do we use this prop?

Reply:
Yes, it's used in the `getDisplayModeInput` and then `computeDisplayModeFromInput`. It's used to determine whether to show the video in the vertical filmstrip tiles

Reply:
Yep I saw it now! Thanks!

Reply:
Actually shouldn't we distinguish in between which filmstrip is the Thumbnail related to?

I see that we use the avatar mode for active participants. Doesn't that mean that both the thumbnails in the main filmstrip and in the stage filmstrip will show avatar?

Reply:
You're right, I'll fix it